src/packer.pkr.hcl

```diff
@@ -40,6 +40,12 @@ variable "release_url" {
   type        = string
 }
 
+variable "packages_dir" {
```

Comment:
It totally slipped my mind but please add this to the list of variables in the README.

Reply:
@mcdonnnj resolved in [7588860](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/75888601ead2944625074bcc667c2e7d19ca8b5b)